build/assets/scripts/low-latency-hooks.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
This is executed for all the pods, not just guaranteed ones, right?

Reply:
Right

Reply:
We need a scale test.. starting 250 pods at the same time and such.